config/default/ValidatorsFactory.conf.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use oat\tao\model\oauth\DataStore;
+use oat\taoLti\models\classes\LtiProvider\RdfLtiProviderRepository;
+use oat\taoLti\models\classes\LtiProvider\Validation\ValidatorsFactory;
+
+return new ValidatorsFactory(
+    [
+        ValidatorsFactory::OPTION_VALIDATORS => [
```

Comment:
IMO we should only allow customization for the customers for things that are environment dependent (DB, ws urls, credentials, paths, etc). But a common form validator should not be the case IMO. Letting the validation configurable also allow the possibility for the customer to insert wrong data in the db, cause they can skip / change the validation at all